File: Wrap-Up/java/ArmstrongNumber.java
public class ArmstrongNumber {

    public static void main(String[] args) {
        int number = 153;
        int originalNumber = number;
        int sumOfCubes = 0;


        while (number > 0) {
            int digit = number % 10;                     
            sumOfCubes = sumOfCubes + (digit * digit * digit); 
            number = number / 10;                        
        }

       
        if (sumOfCubes == originalNumber) {
            System.out.println(originalNumber + " is an Armstrong number.");
        } else {
            System.out.println(originalNumber + " is not an Armstrong number.");
        }
    }
}

File: Wrap-Up/javaScript/armstrongNumber.js
let number = 153;
let originalNumber = number;
let sumOfCubes = 0;


    while (number > 0) {
        let digit = number % 10;                     
        sumOfCubes = sumOfCubes + (digit * digit * digit); 
        number = number / 10;                        
        }

       
        if (sumOfCubes == originalNumber) {
            console.log(originalNumber + " is an Armstrong number.");
        } else {
            console.log(originalNumber + " is not an Armstrong number.");
        }
    
